grade.go

```diff
@@ -32,6 +32,9 @@ type Config struct {
 
 	// HardwareID is a user-specified string to represent the hardware on which the benchmarks have run.
 	HardwareID string
+
+	// Branch is the tag value to use to indicate which branch of the repository was used for the benchmarks that have run.
```

Comment:
The godoc should also clarify that the branch is optional.

Reply:
done